### PR TITLE
UserService

### PR DESCRIPTION
**Pull request by rodionovsasha**

### CODE REVIEW

File: src/main/java/com/github/rodionovsasha/commentservice/entities/User.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import java.io.Serializable;
+
+@NoArgsConstructor
+@Getter @Setter
+@Entity
+public class User implements Serializable {
+    private static final long serialVersionUID = 28116440335967L;
```
Comment (vkorobkov) [inline src/main/java/com/github/rodionovsasha/commentservice/entities/User.java]: How did you generate this `serialVersionUID`?

Comment (rodionovsasha) [reply src/main/java/com/github/rodionovsasha/commentservice/entities/User.java]: I choose some random number

File: src/main/java/com/github/rodionovsasha/commentservice/services/impl/UserServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    public User getActive(long userId) throws InactiveUserException, UserNotFoundException {
+        val user = getById(userId);
+        if (!user.isEnabled()) {
+            throw new InactiveUserException(String.format("User with userId '%d' is not active", userId));
+        }
+        return user;
+    }
+
+    @Override
+    public void updateName(long id, String name) throws UserNotFoundException {
```
Comment (vkorobkov) [inline src/main/java/com/github/rodionovsasha/commentservice/services/impl/UserServiceImpl.java]: Probably we can not change name and age of deactivated user

File: src/main/java/com/github/rodionovsasha/commentservice/services/impl/UserServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        val currentUser = getById(id);
+        currentUser.setName(name);
+        repository.saveAndFlush(currentUser);
+    }
+
+    @Override
+    public void updateAge(long id, int age) throws UserNotFoundException {
+        val currentUser = getById(id);
+        currentUser.setAge(age);
+        repository.saveAndFlush(currentUser);
```
Comment (vkorobkov) [inline src/main/java/com/github/rodionovsasha/commentservice/services/impl/UserServiceImpl.java]: What happens if we use `save` instead `saveAndFlush` ?

Comment (rodionovsasha) [reply src/main/java/com/github/rodionovsasha/commentservice/services/impl/UserServiceImpl.java]: On saveAndFlush, changes will be flushed to DB immediately in this command. With save, this is not necessarily true, and might stay just in memory, until flush or commit commands are issued.

File: src/main/java/com/github/rodionovsasha/commentservice/services/impl/UserServiceImpl.java (hunk 2 above)
Comment (rodionovsasha) [reply src/main/java/com/github/rodionovsasha/commentservice/services/impl/UserServiceImpl.java]: fixed

**Pull request closed without merging**